beacon_node/network/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 /// This crate provides the network server for Lighthouse.
 pub mod error;
+#[allow(clippy::unit_arg)]
```

Comment:
I feel like this would be more natural inside the module, either on the offending code snippet, or as a `#![allow(clippy::unit_arg)]` (the `!` applies the attribute to its enclosing module)